os/include/termios.h

```diff
@@ -358,6 +359,10 @@ int tcsendbreak(int fd, int duration);
  */
 int tcsetattr(int fd, int options, FAR const struct termios *termiosp);
 
+/*
+ * enable/disable the loopback mode.
+ */
+void loopback_en(int fd, int cmd, FAR struct termios *termiosp);
```

Comment:
function name suggest enable but description mentions both enable and disable feature..
can you change the function name